README.md

```diff
@@ -218,6 +219,14 @@ flag take precedence over labels assigned with this flag.
 
 `-disk-size-gb`: The disk size in GB to use when creating the image.
 
+`-oem-size`: The size of the extended OEM partition with unit `G`,`M`,`K` or `B`. 
```

Comment:
Can we also add information like what happens when OEMsize is less than 16MB or disk size is less than `10GB + oem-size`?